app/src/main/java/org/mozilla/tv/firefox/ext/Js.kt

```diff
@@ -91,15 +91,31 @@ var _firefoxTV_isPlaybackStateObserverLoaded;
         javaInterface.syncPlaybackState(isVideoPresent, isPlaying, positionSeconds, playbackRate);
     }
 
-    function adjustVideoCSS() {
+    /*
+     * When a video has left=50% & transform=translateX(-50%) properties (a common CSS centering
+     * trick), videos are pushed outside their containers: if we see similar properties, we remove
+     * them as a workaround. #2526
+     */
+    function removeNegativeTranslationCSSCentering() {
         const rawVideoElements = document.getElementsByTagName('video');
         // Extract elements from HTMLCollection and collect them into an Array
         const videoElements = new Array(...rawVideoElements);
```

Comment:
nit: see previous comment: looks like this wasn't changed.

I realized this syntax is also impossible to google search so it's actually really bad to keep. :P Otherwise I might say it wasn't worth it.

Reply:
Oh geez, sorry.  I swear I fixed that....